VERSION

```diff
@@ -1 +1 @@
-1.10.0
+1.12.0
```

Comment:
we have already 1.17.1. @rnovikovP should it be updated here?

Reply:
@EvgeniyPatlan, for now, we don't have any plans to work on this feature. this was PoC and we blogged about it